x-pack/plugins/apm/public/services/filters/get_encoded_es_filters.ts

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
What about calling the folder `ui_filters` to be more specific. Or is it also other filters?